openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -3,277 +3,124 @@
 """
 from collections import OrderedDict
 
-from django.conf import settings
-from edx_django_utils.monitoring import get_code_owner_from_module
 from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
 from edx_rest_framework_extensions.permissions import IsStaff
-from edx_toggles.toggles import SettingDictToggle, SettingToggle
+from edx_toggles.toggles.state import ToggleStateReport, get_or_create_toggle_response
```

Comment:
See edx-toggles comment about this import, and why it isn’t just from toggles?

Reply:
From the other PR, it sounds like this intentional, so marking this as resolved.  Thanks.